packages/backend/src/panels/AbstractWebviewPanel.ts

```diff
@@ -23,10 +23,13 @@ export abstract class AbstractWebviewPanel {
   protected isInBAS: boolean;
   protected rpc: RpcExtension;
   protected flowPromise: FlowPromise<void>;
+  protected viewColumn: vscode.ViewColumn = vscode.ViewColumn.One;
 
-  public loadWebviewPanel(uiOptions?: unknown): Promise<void> {
+  public loadWebviewPanel(uiOptions?: any): Promise<void> {
     this.disposeWebviewPanel();
-
+    if (uiOptions?.viewColumn && uiOptions.viewColumn in vscode.ViewColumn) {
```

Comment:
please change this condition to this one:

if (uiOptions?.viewColumn in vscode.ViewColumn) {